apps/sample-app/src/App.tsx

```diff
@@ -0,0 +1,68 @@
+import './App.css';
+
+import { Button, FluentProvider, Spinner, teamsLightTheme, Text, Theme } from '@fluentui/react-components';
+import { app, authentication } from '@microsoft/teams-js';
+import React, { useState } from 'react';
+
+import { ProfileContent } from './components/Profile';
+import { appInitializationFailed, getTheme } from './components/utils';
+
+const App: React.FC = () => {
+  const [isInitialized, setIsInitialized] = React.useState(false);
+  const [accessToken, setAccessToken] = React.useState<string>();
+
+  React.useEffect(() => {
+    (async () => {
+      await app.initialize();
+      setIsInitialized(true);
+    })();
+  }, [setIsInitialized]);
+
+  const handle = React.useCallback(async () => {
+    const result = await authentication.authenticate({
+      url: 'https://localhost:4003/?auth=1',
```

Comment:
hardcoding this to `https://localhost:4003` means if we were to deploy it somewhere (like Azure) it would stop working. Can you use relative paths instead? Using `URL` to get one might be a good idea: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL. I haven't tried it, but something like `(new URL('/?auth=1')).toString()` might work